src/main/java/edu/kis/vh/nursery/listAndStack/IntLinkedList.java

```diff
@@ -51,7 +52,7 @@ public int top() {
 	@Override
 	public int pop() {
 		if (isEmpty())
-			return -1;
+			return EMPTY_STACK_ERROR;

```

Comment:
This should refer to the same constant as in IntArrayStack methods (not another constant within IntLinkedList class)